enhancements/etcd/dr-leveraging-ceo-scaling.md

```diff
@@ -0,0 +1,151 @@
+---
+title: cluster-etcd-operator
+authors:
+  - "@alaypatel07"
+reviewers:
+  - "@hexfusion"
+  - "@deads2k"
+  - "@retroflexer"
+approvers:
+ - TBD
+creation-date: 2020-2-21
+last-updated: 2020-2-21
+status: brainstorming
+see-also: 
+replaces:
+superseded-by:
+---
+
+# Disaster Recovery options leveraging Cluster Etcd Operator scaling up
+Currently the Disaster Recovery scripts support 3 scenarios:
```

Comment:
missing loss of quorum

Reply:
added the detail in each scenario below.